webapp/src/constants/index.js

```diff
@@ -11,3 +11,8 @@ export const RHSStates = {
     UNREADS: 'unreads',
     ASSIGNMENTS: 'assignments',
 };
+
+export const JitterForReconnectAPICall = {
+    MAX_TIME: 1,
+    MIN_TIME: 10,
```

Comment:
also, specify the unit of time here in the variable name like `MAX_TIM_IN_MINUTE`